src/sentry/snuba/metrics/query_builder.py

```diff
@@ -444,6 +458,10 @@ def translate_meta_results(
     return sorted(results, key=lambda elem: elem["name"])
 
 
+def coerce_rule_type():
```

Comment:
```suggestion
```
i don't see usage of it.

Reply:
I have removed it, you are too quick that I wasn't able to commit the fix lol